### DISCUSSION TITLE
Fast preview of images directly in the editor as mouseover function

### DISCUSSION THREAD

**Discussion by spechto** [WinEdt-Team/WinEdt-Support#50] [1.3 Ideas and features requests]
Hello everyone,

I often have to include graphics for the creation of documentation, e.g:
`\includegraphics[scale=0.5]{images/image.png}`
Is there a (simple) way to display a preview of the image when hovering over `\includegraphics` or the file name?
I am currently still using WinEdt 10. 

Regards
Martin

**Comment by Alex-Simonic** [comment]
WinEdt currently doesn't have this functionality. Maybe one day...

**Comment by spechto** [comment]
this functionality would be a good reason to buy an upgrade :-)
